### DISCUSSION TITLE
Two Semi-Issues Since Updating

### DISCUSSION THREAD

**Discussion by MonfGeiger** [black7375/Firefox-UI-Fix#752] [Q&A] [answered]
I recently updated my lepton photon ui to the latest available release at the time of this post (7.4.0), and ofc i updated my firefox to the latest stable 64 bit release (116) days earlier. idk if its from the older lepton photon, or something firefox changed itself, but im running into two weird UI issues. First, im getting double separator lines in certain menus, most notibly my right click menu, and quite a few of the top left menu boxes (file, edit, view, tools, & help)
![Capture](https://github.com/black7375/Firefox-UI-Fix/assets/28603306/f0b9536c-cdb3-41f8-817e-8b2f02c49693)
I would like to know if i can edit some line of code in one of the .css files that would fix this, or if what i need to change is in some about:config setting instead

Second, i noticed that my dynamic separator boxes have had their grey line colors changed to total dark as in the second pic here
![Capture2](https://github.com/black7375/Firefox-UI-Fix/assets/28603306/fd05a8c0-0738-46ac-becc-35acc418067e)
any chance i can change them back to a lighter but almost un-noticable grey?

**Comment by Pukkels** [comment]
I have same issue as you, as a test i removed the ui-fix files and ff is ok.
But bottom favo menu scrolls of my screen, thats why i used the ui-fix in first place.
i use ff 116 portable under win 11 22h2 (X64) dutch language.

**Comment by black7375** [comment]
1. Context menu double separator fixed at #732
Try to new release https://github.com/black7375/Firefox-UI-Fix/releases/tag/v7.5.0
2. Are you using `userChrome.tab.dynamic_separator`? I'll check it.

**Reply by MonfGeiger** [reply]
the new release fixed both the double separator context issue, and the second pic thing. also yes, i am using the dynamic separator. apparently when i put the user.js file in to potentially update any newer menu line options, it activated both the dynamic separator and static separator. the dynamic separator is what im using currently, and the dynamic separator lines are total black, like AMOLED Dark

i just need to figure out how to change the colors to grey and im set

**Reply by MonfGeiger** [reply, marked as answer]
GOT IT, took me a bit of digging, but i had to do this to change the color from all black to darkish grey
```
 /* Navbar Border */
  #navigator-toolbox:-moz-lwtheme {
    **--tabs-border-color: rgba(160, 160, 160, 0.4)**; /* Legacy: v96, (0, 0, 0, 0.3) -> (0, 0, 0, 0.4)  */
    --lwt-tabs-border-color: rgba(0, 0, 0, 0.4);
  }
```
just have to make sure that it isnt set to anything else and not to overwrite my userchrome file, and im set. thanks for the help guys

**Reply by black7375** [reply]
If both `userChrome.tab.dynamic_separator` and `userChrome.tab.static_separator` are active, problems can occur.
If you use `userChrome.tab.dynamic_separator`, set `userChrome.tab.static_separator` to `false`.

**Reply by Pukkels** [reply]
As "MonfGeiger" already said, Version 7.50 solved the issues, both "userchrome.tab values were set to false!.

Thanks for fixing!.
